src/App.tsx

```diff
@@ -17,18 +80,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                handleQuery={handleQuery}
+                handleStatus={handleStatus}
+                handleEraseInput={handleEraseInput}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {filteredTodos.length
```

Comment:
Let's add some new boolean `useState` like `isLoading` and use it here.
Initial this `isLoading` will be true and after you get response from the server you will change it to false